release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,1534 @@
+creates user-defined cluster objects. Build, configure, and run Cluster Loader
+to measure performance metrics of your {product-title} deployment at various
+cluster states.
+
+[[ocp-37-scale-benefits-of-using-the-overlay-graph-driver]]
+==== Guidance on Overlay Graph Driver with SELinux
+
+In {product-title} 3.7, guidance about the
+xref:../scaling_performance/optimizing_storage.adoc#benefits-of-using-the-overlay-graph-driver[benefits
+of using the Overlay Graph Driver with SELinux] is now available.
+
+[[ocp-37-scale-providing-storage-to-an-etcd-node-using-pci-passthrough-with-openstack]]
+==== Providing Storage to an etcd Node Using PCI Passthrough with OpenStack
+
+Guidance on
+xref:../scaling_performance/host_practices.adoc#providing-storage-to-an-etcd-node-using-pci-passthrough-with-openstack[Providing
+Storage to an etcd Node Using PCI Passthrough with OpenStack] is now available.
+
+[[ocp-37-networking]]
+=== Networking
+
+[[ocp-37-network-policy]]
+==== Network Policy
+Network Policy is now fully supported in {product-title} 3.7.
+
+Network Policy is an optional plug-in specification of how selections of pods
+are allowed to communicate with each other and other network endpoints. It
+provides fine-grained network namespace isolation using labels and port
+specifications.
+
+After installing the Network Policy plug-in, an annotation that flips the
+namespace from `allow all traffic` to `deny all traffic` must first be set on
+the namespace. At that point, `NetworkPolicies` can be created that define what
+traffic to allow. The annotation is as follows:
+
+----
+$ oc annotate namespace ${ns} 'net.beta.kubernetes.io/network-policy={"ingress":{"isolation":"DefaultDeny"}}'
+----
+
+The allow-to-red policy specifies "all red pods in namespace `project-a` allow
+traffic from any pods in any namespace." This does not apply to the red pod in
+namespace `project-b` because `podSelector` only applies to the namespace in
+which it was applied.
+
+.Policy applied to project
+----
+kind: NetworkPolicy
+apiVersion: extensions/v1beta1
+metadata:
+  name: allow-to-red
+spec:
+  podSelector:
+    matchLabels:
+      type: red
+  ingress:
+  - {}
+----
+
+See
+xref:../admin_guide/managing_networking.adoc#admin-guide-manage-networking[Managing
+Networking] for more information.
+
+[[ocp-27-cluster-ip-range-more-flexible]]
+==== Cluster IP Range Now More Flexible
+
+Cluster IP ranges are now more flexible by allowing multiple subnets for hosts.
+This provides the capability to allocate multiple, smaller IP address ranges for
+the cluster. This makes it easier to migrate from one allocated IP range to
+another.
+
+There are multiple comma-delimited CIDRs in the configuration file. Each node is
+allocated only a single subnet from within any of the available ranges. You can
+not allocate different-sized host subnets, or use this to change the host subnet
+size The `clusterNetworkCIDRs` can be different sizes, but must be equal to or
```

Comment:
Nodes could allocate different-sized subnets by setting different hostSubnetLength
Lack of period between `size The`